src/pro/javacard/gp/GPTool.java

```diff
@@ -429,6 +441,11 @@ public static void main(String[] argv) throws Exception {
 						}
 					}
 
+					// list access rules
+					if (args.has(OPT_ACR_LIST)) {
+						gp.acrList(card);
```

Comment:
As the connection must not be authenticated, it might be better to keep it outside of the main GlobalPlatform class (that could be renamed GlobalPlatforCard at some stage, for clarity) ?